<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProposalCreationDialog</class>
 <widget class="QDialog" name="ProposalCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>418</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>418</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>660</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proposal Creation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../blocknetdx.qrc">
    <normaloff>:/icons/proposalcreation</normaloff>:/icons/proposalcreation</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="preparegroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>201</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Prepare Proposal</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="10,10,0,0,0,0" columnminimumwidth="100,100,0,0,0,0">
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="4" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Payment Count</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Block Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>BlocknetDX Address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Monthly Payment  BlocknetDX</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="blockstarttextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QTextEdit" name="paymentcounttextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QTextEdit" name="monthlyblockdxtextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="QTextEdit" name="nametextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="5">
      <widget class="QTextEdit" name="urltextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="undoRedoEnabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="QTextEdit" name="blocknetdxaddresstextEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="preparepushButton">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>160</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Prepare</string>
    </property>
   </widget>
   <widget class="QLabel" name="costlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preparation/Submission cost: 50 Block</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="submitgroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>661</width>
     <height>151</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>Submit Proposal</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fee TX</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="feetxtextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>591</width>
      <height>16</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="submitpushButton">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>110</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="closebuttonBox">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>380</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="statuslabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>365</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Current Status: Normal</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../blocknetdx.qrc"/>
 </resources>
 <connections/>
</ui>
